### PR TITLE
Changes made by Haidra-org before deployment

### PR DESCRIPTION
**Pull request by tazlin**
This PR is to help keep your original repo in sync. If you source any PRs back, there should be no merge conflict and your existing actions in your repo should therefore not interfere with our CI/CD scheme.

- Supports the AI-Horde `/news/` endpoint to populate the new items
- Supports the new `/document/terms` endpoint
- Hamburger menu for mobile now works
- Slow API calls are now non-blocking.
- Support hosting on other than `/` 
- Certain links now rely on [AI-Horde](https://github.com/Haidra-Org/AI-Horde) to handle, such as `/register`.
- Various style tweaks
- Changed the logo to the haidra-org logo for now.

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
                   class="inline-flex items-center p-2 ml-1 text-sm text-gray-500 rounded-lg lg:hidden hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-gray-200 dark:text-gray-400 dark:hover:bg-gray-700 dark:focus:ring-gray-600"
-                  aria-controls="mobile-menu-2" aria-expanded="false">
+                  aria-controls="mobile-menu-2" aria-expanded="false"
+                  (click)="toggleMobileMenu(); $event.stopPropagation()"
+                  >
             <span class="sr-only">{{'open_main_menu' | transloco}}</span>
-            <inline-svg href="/assets/img/open-menu.svg" />
-            <inline-svg href="/assets/img/close-menu.svg" />
+            <inline-svg href="assets/img/open-menu.svg" />
+            <inline-svg href="assets/img/close-menu.svg" />
```
Comment (RikudouSage) [inline src/app/app.component.html:17]: Why this change? It's throughout the whole PR.

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,34 +1,39 @@
...
+          <ul class="flex flex-col mt-4 font-medium border-b border-gray-300 lg:border-0 lg:flex-row lg:space-x-8 lg:mt-0 bg-gray-900">
             <li>
-              <a routerLink="/" class="block py-2 pl-3 pr-4 text-white bg-purple-700 rounded lg:bg-transparent lg:text-purple-700 lg:p-0 dark:text-white" aria-current="page">{{'home' | transloco}}</a>
+              <a routerLink="/" class="block py-2 pl-3 pr-4 text-gray-500 bg-gray-400 rounded lg:bg-transparent lg:text-gray-500 lg:p-0 dark:text-white" aria-current="page">{{'home' | transloco}}</a>
             </li>
             <li>
-              <a routerLink="/register" class="block py-2 pl-3 pr-4 text-white bg-purple-700 rounded lg:bg-transparent lg:text-purple-700 lg:p-0 dark:text-white" aria-current="page">{{'register_account' | transloco}}</a>
+              <a href="/register" class="block py-2 pl-3 pr-4 text-gray-500 bg-gray-400 rounded lg:bg-transparent lg:text-gray-500 lg:p-0 dark:text-white" aria-current="page">{{'register_account' | transloco}}</a>
```
Comment (RikudouSage) [inline src/app/app.component.html:26]: Why this? We're not implementing registration before releasing it?

File: src/app/pages/homepage/parts/quickstart/homepage-quickstart.component.html [hunk 3]
```diff
@@ -6,18 +6,18 @@ <h2 class="mb-6 text-3xl font-extrabold tracking-tight text-gray-900 dark:text-w
           <transloco key="quickstart" />
         </h2>
 
-        <p class="mb-4">
-          <transloco key="quickstart.register_account" [params]="{registerLink: {route: '/register'}}" />
+        <p class="mb-4" [innerHTML]="'quickstart.register_account' | transloco" >
+
```
Comment (RikudouSage) [inline src/app/pages/homepage/parts/quickstart/homepage-quickstart.component.html:10]: This would be better reverted and the `registerLink` changed from `{route: '/register'}` to just `/register`, that way it will use href instead of routerLink.

File: src/app/app.component.html (hunk 1 above)
Comment (tazlin) [reply src/app/app.component.html:17]: The mobile experience hamburger menu did not seem to work, or if it did, it was in a non obvious way. This was my effort at making it work and work with a click away function to hide it

Comment (tazlin) [reply src/app/app.component.html:17]: Correct. We'll tackle this another time.

File: src/app/app.component.html (hunk 2 above)
Comment (tazlin) [reply src/app/app.component.html:26]: Correct, we'll resolve this another time.

File: src/app/app.component.html (hunk 1 above)
Comment (tazlin) [reply src/app/app.component.html:17]: Sorry, I misunderstood. This was so the site isn't bound to /, and would work on subdirs

Comment (RikudouSage) [reply src/app/app.component.html:17]: I meant the change that removes the leading `/` throughout the whole PR, sometimes replacing it with `./`

File: src/app/pages/homepage/parts/quickstart/homepage-quickstart.component.html (hunk 3 above)
Comment (tazlin) [reply src/app/pages/homepage/parts/quickstart/homepage-quickstart.component.html:10]: If you can demonstrate it works the same, that sounds fine to me

File: src/app/app.component.html (hunk 1 above)
Comment (tazlin) [reply src/app/app.component.html:17]: Yes sorry, I realized after. I answered above

Comment (RikudouSage) [reply src/app/app.component.html:17]: Ah, nevermind, I haven't read the comment that came with the PR. Anyway, it would be better to either stick to `./` or no prefix, not mixing it up.

Comment (tazlin) [reply src/app/app.component.html:17]: That's fair.
